### PR TITLE
Add a bunch more specs to biblio.json

### PR DESCRIPTION
**Pull request by foolip**
These are from links that appear in Chromium's source code, although all of
these might not actually be implemented in Chromium.

### CODE REVIEW

File: biblio.json [hunk 1]
```diff
@@ -1,35 +1,71 @@
...
-		"title": "Feature Policy" 
+		"title": "Feature Policy"
 	},
 	"GEOLOCATION-SENSOR": {
 		"href": "https://wicg.github.io/geolocation-sensor/",
 		"title": "Geolocation Sensor"
 	},
+	"INPUT-DEVICE-CAPABILITIES": {
+		"href": "https://wicg.github.io/InputDeviceCapabilities/",
+		"title": "InputDeviceCapabilities"
```
Comment (foolip) [inline biblio.json]: @RByers, WDYT about changing the title of the spec to have spaces? The only other spec name I'm aware of that is directly the name of the API is https://w3c.github.io/editing/execCommand.html

**Review by marcoscaceres — APPROVED** [review]
Tentative approval from me pending outcome of INPUT-DEVICE-CAPABILITIES

File: biblio.json (hunk 1 above)
Comment (foolip) [reply biblio.json]: I've sent https://github.com/WICG/InputDeviceCapabilities/pull/29 and will tentative change it here too.

Comment (RByers) [reply biblio.json]: SGTM, thank you.  Merged.

**Comment by foolip** [comment]
Thanks @RByers, over to @marcoscaceres for merge.

**Pull request merged**